docs/locale/en_pseudo/LC_MESSAGES/index.po

```diff
@@ -0,0 +1,41 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2016-2017, Open Data Services
```

Comment:
Should the `copyright` also be updated in here?

Reply:
I think this whole file can probably just be removed.

Reply:
@Bjwebb yeah, probably. Shall I just remove `docs/locale`?

Reply:
Yes, if there's no translations imminent.